docs/source/io_formats/mgxs_library.rst

```diff
@@ -8,297 +8,155 @@ OpenMC can be run in continuous-energy mode or multi-group mode, provided the
+:Attributes: - **awr** (*double*) -- The atomic weight ratio (optional, i.e. it
+               is not meaningful for material-wise data)
+             - **fissionable** (*int*) -- Whether the dataset is fissionable
+               (1) or not (0).
+             - **representation** (*char[]*) -- The method used to generate and
+               represent the multi-group cross sections.  That is, whether they
+               were generated with scalar flux weighting (or reduced to a
+               similar representation) and thus are angle-independent, or if the
+               data was generated with angular dependent fluxes and thus the
+               data is angle-dependent.  Valid values are either "isotropic" or
+               "angle".
+             - **num-azimuthal** (*int*) -- Number of equal width angular bins
+               that the azimuthal angular domain is subdivided if the
+               `representation` attribute is "angle". This parameter is
+               ignored otherwise.
+             - **num-polar** (*int*) -- Number of equal width angular bins
+               that the polar angular domain is subdivided if the
+               `representation` attribute is "angle". This parameter is
+               ignored otherwise.
+             - **scatter-format** (*char[]*) -- The representation of the
+               scattering angular distribution.  The options are either
+               "legendre", "histogram", or "tabular".  If not provided, the
+               default of "legendre" will be assumed.
+             - **order** (*int*) -- Either the Legendre order, number of bins,
+               or number of points (depending on the value of `scatter-format`)
+               used to describe the angular distribution associated with each
+               group-to-group transfer probability.
+
+**/<library name>/kTs/**
+
+:Datasets: - **<TTT>K** (*double*) -- kT values (in MeV) for each Temperature
+             TTT (in Kelvin), rounded to the nearest integer
+
+**/<library name>/<TTT>K/**
+
+Temperature-dependent data, provided for temperature <TTT>K.
+
+:Datasets: - **total** (*double[]* or *double[][][]*) -- Total cross section.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].
+           - **absorption** (*double[]* or *double[][][]*) -- Absorption
+             cross section.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].
+           - **fission** (*double[]* or *double[][][]*) -- Fission
+             cross section.
+             This is a 1-D vector if `representation` is "isotropic", or a 3-D
+             vector if `representation` is "angle" with dimensions of
+             [groups, azimuthal, polar].  This is only required if the data set
```

Comment:
This is just me being OCD, but might not "dataset" be more appropriate than "data set"? This would better correspond to the spelling used by HDF5, as well as the broader "data science" community.